docs/pt/docs/fastapi-people.md

```diff
@@ -0,0 +1,137 @@
+# FastAPI Pessoal
```

Comment:
It doesn't sound as cool as in English, what do you think about: `Agentes FastAPI`? :sunglasses: